run.py

```diff
@@ -16,37 +18,11 @@ def setup_logger(level) -> logging.Logger:
         Logger: root set up for console and file logging
     '''
 
-    logger = logging.getLogger()
-    logger.setLevel(logging.DEBUG)
-    
-    # Primary handlers. Levels can be set with --level option
```

Comment:
Consider readding a level, especially for console logging, for cases where we only want to see critical and error logging, not absolutely everything.